example/src/App.tsx

```diff
@@ -20,6 +20,8 @@ import LandscapeFullscreenHandling from './screens/LandscapeFullscreenHandling';
 import SystemUI from './screens/SystemUi';
 import OfflinePlayback from './screens/OfflinePlayback';
 import Casting from './screens/Casting';
+import LockScreenControls from './screens/LockScreenControls';
```

Comment:
I guess we should not use the naming "Lock screen controls" anymore to stay consistent with the API naming. Would be good to rename every occurrence in the whole code base (incl. file names)

Reply:
Address in b97c8c1eb5b660354be1b087db575c04ff51321c